project/src/main/webapp/event-details.js

```diff
@@ -88,16 +87,18 @@ function getLinkForOpportunity(opportunityId) {
 function getEventDetails() {
   const queryString = window.location.search;
   const urlParams = new URLSearchParams(queryString);
+  const eventId = urlParams.get('eventId');
   fetch(
       '/create-event?' +
-      new URLSearchParams({'eventId': urlParams.get('eventId')}))
+      new URLSearchParams({'eventId': eventId}))
       .then((res) => (res.json())).then((data) => {
```

Comment:
Can use async and await style here for example:
Make it an async function and use
const res = await fetch(...)
cost data = await res.json()